static/src/javascripts/projects/commercial/modules/user-features.spec.js

```diff
@@ -174,7 +187,7 @@ describe('The isInBrexitCohort getter', () => {
     it('Is false if the user is logged out', () => {
         jest.resetAllMocks();
         isUserLoggedIn.mockReturnValue(false);
-        expect(isAdFreeUser()).toBe(false);
+        expect(isInBrexitCohort()).toBe(false);
```

Comment:
@lmath I think you were the last to work on this test. I assume this is what you'd intended?